src/main/java/core/basesyntax/converter/FileReaderImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.converter;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(filePath))) {

```

Comment:
You are using `java.io.FileReader` here. It's better to import this class at the beginning of the file to make the code cleaner and easier to read.